### ISSUE TITLE
make tidy.SummarizedExperiment exportable or add to biobroom

### ISSUE THREAD

**Issue by hummuscience** [pkimes/upbm#12]
I just stumbled onto your tidy() function for SummarizedExperiment objects and I liked it a lot. Specially for the ability to actually output a long format when the colData needs to be present.

Actually, it is a better implementation than the biobroom implementation. I was wondering whether it is possible to either:

1) make it exportable so people can import it in their packages
or (following the bioconductor vision)
2) Suggest a change to the biobroom package to introduce your changes

All the best!

**Comment by pkimes** [comment]
@Cumol thanks - I'm glad you found the _tidy_ function useful. I also found the _biobroom_ implementation lacking (why I wrote this). I'm a bit backed up right now, but will consider making a pull request to _biobroom_. 

It looks like the _biobroom_ implementation already allows colData to be added with the `addPheno=` option and outputs values as long format. The only benefits I see in my implementation are [1] allowing wide format, [2] allowing rowData by default, [3] allowing multiple assays at once.

**Comment by Cumol** [comment]
True, I also found that it is capable of adding colData. But you are right, it is missing the additional options and in my case, adding rowData is crucial.

I will use your function for now and keep an eye  biobroom if you ever think about adding that functionality (maybe alls as options like addRow = TRUE etc.)
